doc/servermd/RESTAPI.md

```diff
@@ -131,6 +131,7 @@ Data Model:<br>
       transcoding: object(Transcoding),    // the transcoding control
       notifying: object(Notifying),        // notification control
       selectActiveAudio: boolean,          // select 3 most active audio streams for the room
+      senderBandwidthEstimation: boolean,  // enable send side bandwidth estimation
```

Comment:
I think this should be part of webrtc agent configuration instead of room configuration.

Reply:
After discussion, ok to put this as part of room configuration.

Reply:
Change its name to `webrtcSenderSideBandwidthEstimation`? This configuration doesn't work for WebTransport connections.